realwork/realwork/SignUpBoard/SignUpVC.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  SignUpVC.swift
+//  realwork
+//
+//  Created by Nathan Ngobale on 2020/03/19.
+//  Copyright © 2020 Nathan Ngobale. All rights reserved.
+//
+
+import UIKit
+//import Firebase
+import NLibrary
+
+class SignUpVC: UIViewController {
+
+    @IBOutlet weak var edtEmail: UITextField!
+    @IBOutlet weak var edtPassword: UITextField!
+    @IBOutlet weak var loaderIndicator: UIActivityIndicatorView!
+    lazy var userVM: UserVM = {
```

Comment:
I don't think it is necessary to create a lazy var in this way. I think the following should work: 
lazy var userViewModel = UserVM(viewController: self, userRepo UserRepo())

Also, is a lazy init required ? I see that this ViewModel is instantiated again in the viewDidLoad.

Reply:
Made the necessary change you required.